testing/mock/mock.go

```diff
@@ -178,7 +180,11 @@ func (am AppModule) OnChanOpenTry(
 }
 
 // OnChanOpenAck implements the IBCModule interface.
-func (am AppModule) OnChanOpenAck(sdk.Context, string, string, string) error {
+func (am AppModule) OnChanOpenAck(ctx sdk.Context, portID string, channelID string, counterpartyVersion string) error {
+	if am.Middleware.OnChanOpenAck != nil {
+		return am.Middleware.OnChanOpenAck(ctx, portID, channelID, counterpartyVersion)
+	}
```

Comment:
would be added to every mock callback, ie, use developer provided function or do default logic